### ISSUE TITLE
Drop a Datasource

### ISSUE THREAD

**Issue by tungnt1203** [mindsdb/minds_python_sdk#24]
I am currently working on developing a JS SDK. In my implementation, I have a list of "minds," and within each mind, there is a random datasource.

The issue occurs when I drop a datasource and then reload the page at https://mdb.ai/minds. At this point, the error appears.
<img width="943" alt="image" src="https://github.com/user-attachments/assets/a6288289-2fcc-4498-972e-9805751d6fa0">

**Comment by martyna-mindsdb** [comment]
Hi @tungnt1203 
Please use this endpoint to drop a data source - https://docs.mdb.ai/docs/api/datasources#delete-datasource
It shouldn't have any params in the body.

**Comment by tungnt1203** [comment]
@martyna-mindsdb  Yes, I used the API as you mentioned. However, this datasource is linked with Minds, and when I delete it, it causes an error on the Minds side.

**Comment by chandrevdw31** [comment]
@tungnt1203 have you tried deleting the minds first and then the datasource?

**Comment by ZoranPandovski** [comment]
This is a frontend issue. It should still show the mind even if the data sources were deleted @mpacheco12

**Comment by ZoranPandovski** [comment]
This repository is for the Python SDK so we may transfer this issue to the frontend repo
